src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Microsoft.AspNetCore.Razor.LanguageServer.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultNetCoreWindowsTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(DefaultNetCoreTargetFrameworks)</TargetFrameworks>
```

Comment:
You should add this project to `Razor.Slim.slnf` if you want it to run in CI on Linux/mac.

Reply:
Good callout. I'm currently splitting up the windows and non-windows required tests into two projects. Once that's done I'll add to Razor.slim.slnf